core/src/trezor/ui/layouts/caesar/__init__.py

```diff
@@ -939,15 +940,22 @@ def confirm_total(
 
 if not utils.BITCOIN_ONLY:
 
-    def confirm_ethereum_unknown_contract_warning() -> Awaitable[None]:
+    def confirm_ethereum_unknown_contract_warning(
+        _title: str | None,
+    ) -> Awaitable[None]:
         return show_danger(
             "unknown_contract_warning",
             TR.words__know_what_your_doing,
             title=TR.ethereum__unknown_contract_address,
         )
 
+    def ethereum_address_title() -> str:
```

Comment:
Can we maybe try to fix the inconsistency in the design and thus get rid of these otherwise not so nice looking functions?